src/main/webapp/scripts/toggle.js

```diff
@@ -26,11 +26,16 @@ async function loadContentSection() {
   } else {
     keywords.forEach(loadKeywordSection);
   }
+  const address = encodeURI(getCookie('address'));
```

Comment:
How are we going to make sure this address is for the US?

Reply:
Nevermind, this is related to issue #79. I'll fix the civic servlet on another PR.